lib/secure-boot-lib.robot

```diff
@@ -236,7 +236,64 @@ Check Secure Boot In Windows
     ...    Should Contain    ${out}    True
     RETURN    ${sb_status}
 
-Check Gf Attempt Secure Boot Can Be Selected
+Enable Secure Boot
+    ${setup_menu}=    Enter Setup Menu Tianocore And Return Construction
+    ${device_mgr_menu}=    Enter Submenu From Snapshot And Return Construction
+    ...    ${setup_menu}
+    ...    Device Manager
+    ${secure_boot_menu}=    Enter Submenu From Snapshot And Return Construction
+    ...    ${device_mgr_menu}
+    ...    Secure Boot Configuration
+    Select Enable Secure Boot Option    ${secure_boot_menu}
+    Save Changes And Reset    2
+
+Disable Secure Boot
+    ${setup_menu}=    Enter Setup Menu Tianocore And Return Construction
+    ${device_mgr_menu}=    Enter Submenu From Snapshot And Return Construction
+    ...    ${setup_menu}
+    ...    Device Manager
+    ${secure_boot_menu}=    Enter Submenu From Snapshot And Return Construction
+    ...    ${device_mgr_menu}
+    ...    Secure Boot Configuration
+    Clear Attempt Secure Boot Option
+    Save Changes And Reset    2
+
+Enter Custom Secure Boot Options
+    [Documentation]    Sets the Secure Boot Mode to Custom Mode from
+    ...    the Secure Boot Configuration menu and enters
+    ...    the Custom Secure Boot Options menu
+    ...
+    ${sb_menu}=    Reenter Menu And Return Construction
+    ${is_standardmode}=    Run Keyword And Return Status
+    ...    Should Contain Match    ${sb_menu}    *Standard*
+    IF    ${is_standardmode}
```

Comment:
If possible, we should avoid hard-coding menu positions and use menu construction lists to select entries instead. Kwd for entering advanced menu is already here: https://github.com/Dasharo/open-source-firmware-validation/pull/95/files#diff-b06ff90d8f021dfa2e18d217f3d4c1e6861d6f671893cf292b6718b80227921aR49

While just selecting certain option from the lit is as as simple as: ` Set Option State    ${sb_menu}    Secure Boot Mode    Custom Mode`

This kwd should take care of returning selectable entries only: https://github.com/Dasharo/open-source-firmware-validation/pull/95/files#diff-b06ff90d8f021dfa2e18d217f3d4c1e6861d6f671893cf292b6718b80227921aR9 
With the new many layout it is a bit easier to do.

Reply:
@macpijan that is very useful I was not aware that there is code that handle that. I will try to use that code.

Reply:
That was in my MR (submitted before this one), which means you have probably not looked at that before submitting this. I specifically drafted a plan here: https://github.com/Dasharo/open-source-firmware-validation/pull/95#issue-1984603821 and pointed to: `Make sure to review/rework all kwds in secure boot lib - see this comment: https://github.com/Dasharo/open-source-firmware-validation/pull/95/files#diff-b06ff90d8f021dfa2e18d217f3d4c1e6861d6f671893cf292b6718b80227921aR107 - put useful/tested kwds above that comment, and eventually remove it` to indicate which keywords I see as of a reasonable quality, and which not

Reply:
> That was in my MR (submitted before this one), which means you have probably not looked at that before submitting this. 

@macpijan it is very hard to follow your process. I'm sorry, but I'm not able to sync in 2-3 hours with everything you publish. Changes come in big chunks and make previous assumptions obsolete. It is very hard to make meaningful contributions working that way. It is definitely my fault that I didn't notice the comment you pointed to. I essentially spent most of my contribution time trying to understand the changes you introduced, and the approach was so different from the previous set of keywords that I was lost. Rebasing branches on which someone else works is damaging.